src/site/pages/accessibility-guide.md

```diff
@@ -0,0 +1,86 @@
+# Accessibility Guide
+
+*Please note, this guide is a work in progress and will be updated regularly. We welcome your comments and feedback.*
+
+The goal of software accessibility is to remove barriers to the use of technology so that it can be inclusive of everyone, regardless of physical ability. Accessibility addresses permanent to temporary disabilities and includes a range of visual impairments, as well as physical, sensory or learning challenges, including those that occur normally as adults age. Designing Red Hat software with this broad range of accessibility in mind will empower any user to accomplish their goals with our technology.
+
+Since accessibility is best achieved when it's considered at the beginning, we ask designers and developers who contribute or consume patternfly to understand how accessibility needs can be met. The following is a guide with some techniques on how to design, develop, and test UI to ensure that everyone has a good user experience.
+
+- [Understanding Users’ Needs](#understanding-users-needs)
+- [Checklists](#checklists)
+  - [Design and Development Guidelines](#design-and-development-guidelines)
+      - [What PatternFly Designers and Developers Should Address](#what-patternfly-designers-and-developers-should-address)
+      - [What Product Developers and Designers Should Address](#what-product-developers-and-designers-should-address)
+  - [Testing](#testing)
+  - [Screen Reader Support](#screen-reader-support)
+- [Methods](#methods)
+
+## Understanding Users’ Needs
+
+As with any design, it’s important to understand the user you’re designing for. To develop an inclusive design, it’s important to understand the different needs of [different users](https://a11yproject.com/posts/myth-accessibility-is-blind-people/), and consider the assistive tools and methods they use. The following are some of the ways we can help address these different needs.
+
+### No Vision
+
+Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements on the page, like headers, links, or form elements. Using semantic elements and ensuring that these elements are still meaningful when pulled out of context is critical for the usability of the site.
+
+### Low Vision
+
+Users with low vision can have different needs depending on their eye condition. Needs can range from not relying on color to communicate information, color palettes that provide sufficient contrast, and a layout that is responsive when the user zooms in to magnify the contents of the screen.
```

Comment:
"Eye condition" sounds a little odd to me. How about "depending on the nature of their visual impairment." ? Also, the next sentence is complicated - how about something like "Users may have difficulty with color differentiation, blurriness, or lack of vision in central or peripheral areas. These needs mean that interfaces should not rely on color to communicate information, palettes need to have sufficient contrast, and screens should behave well when a screen magnifier is used."